lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -52,9 +52,13 @@ public function createDatabase($database)
         $params = $this->_conn->getParams();
         $driver = $params['driver'];
         $options = array(
-            'driver' => $driver,
-            'path' => $database
+            'driver' => $driver
         );
+        if(isset($params['memory']) && $params['memory']){
```

Comment:
Sorry got something messed up here. I think your patch is wrong anyways because you could create a persistent database from a current in-memory connection:

1. connect to in-memory DB
2. with the schema manager and that connection create another database that is persistent.

So in this case it is totally unrelated whether you are currently connected to a in-memory database. But what we could do i check whether the given `$database` parameter is `:memory:` and then perform a noop.